redis/src/cluster_routing.rs

```diff
@@ -397,7 +397,12 @@ fn base_routing(cmd: &[u8]) -> RouteBy {
         | b"XGROUP SETID"
         | b"XINFO CONSUMERS"
         | b"XINFO GROUPS"
-        | b"XINFO STREAM" => RouteBy::SecondArg,
+        | b"XINFO STREAM"
+        | b"ZDIFF"
+        | b"ZINTER"
+        | b"ZINTERCARD"
+        | b"ZMPOP"
```

Comment:
```suggestion
        | b"XINFO STREAM"
        | b"SINTERCARD"
        | b"ZDIFF"
        | b"ZINTER"
        | b"ZINTERCARD"
        | b"ZMPOP"
        | b"BZMPOP"
```

`SINTERCARD` and `BZMPOP`

Reply:
@aaron-congo @Yury-Fridlyand 
Are you intending to add all the missing commands that have 2nd arg as a key or some specific commands?

Lets make it exhaustive (if you already touching this code) - please implement a script that parses all the jsons in  $REDIS_ROOT/src/commands/ and filters the commands that have "key" object at index 1 of "arguments" block, per example of ZDIFF:
`"arguments": [
            {
                "name": "numkeys",
                "type": "integer"
            },
            {
                "name": "key",
                "type": "key",
                "key_spec_index": 0,
                "multiple": true
            },
            {
                "name": "withscores",
                "token": "WITHSCORES",
                "type": "pure-token",
                "optional": true
            }
        ]
`
Run this script on the ValKey (https://github.com/valkey-io/valkey/releases/tag/7.2.5) source, so not tot touch the licensed Redis.

Please add this script to the PR - it will form the basis for automatically generate all these hard-coded cases during the compilation in the future

Reply:
I've added SINTERCARD to SecondArgAfterKeyCount and BZMPOP to ThirdArgAfterKeyCount